utils/local-deployment/local-setup.sh

```diff
@@ -20,17 +20,29 @@ set -euo pipefail
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
+echo "setting up kuadrant for provider ${PROVIDER}"
+
 export KUADRANT_NAMESPACE="kuadrant-system"
 export KIND_CLUSTER_NAME="kuadrant-local"
 
 ${SCRIPT_DIR}/deploy-kuadrant-deps.sh
 
-echo "Building kuadrant"
+echo "Building kuadrant ${PROVIDER}"
```

Comment:
this should go in `deploy-kuadrant-deps.sh`

The reason is that local-setup.sh is mean for deploying the controller as a pod. 

But for dev testing, sometimes it is handy to run it locally (outside the cluster), and the kuadrant infra (istio, limitador, authorino) is still needed to be there, so `deploy-kuadrant-deps.sh` is being reused for both deployment modes